test/util/nightwatch-commands/axeCheck.js

```diff
@@ -19,16 +19,16 @@ export function command(context, config, _callback) {
   }, [axeSource]);
 
   // Run axe checks and report
-  this.executeAsync((innerContext, done) => {
+  this.executeAsync((innerContext, rules, done) => {
```

Comment:
It could also be that the one array represents all the arguments between [the function being invoked asynchronously] and [the `done` callback]. The [docs](https://github.com/nightwatchjs/nightwatch-docs/blob/master/api/method/executeAsync.md) aren't that clear about the additional arg.